app/serializers/device_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class DeviceSerializer < ActiveModel::Serializer
+  attributes *%i(id location location_name payload created_at updated_at)
+
+  def location
```

Comment:
Looks like I was wrong on the location bit https://github.com/Safecast/safecastapi/issues/355 - Glad we're keeping that data handy.